package.json

```diff
@@ -9,8 +9,10 @@
         "test:unit": "vue-cli-service test:unit"
     },
     "dependencies": {
+        "cheerio": "^1.0.0-rc.3",
```

Comment:
what is the reason for adding in cheerio and sanitize? If it is not required, can we remove these additions? Let's make sure package-lock.json is also back to before these were added.

Reply:
It's actually because of the `wp-content` component that is used in this component. It's used to render oEmbed codes (YouTube, Vimeo etc) from WordPress. So in this case, these are needed.